lua/bufferline.lua

```diff
@@ -39,7 +39,7 @@ local DEFAULT_OPTIONS = {
 --------------------------
 
 --- Create and reset autocommand groups associated with this plugin.
---- @return number bufferline, number bufferline_update
+--- @return integer bufferline, integer bufferline_update
```

Comment:
Sumneko has become extremely particular about `integer` and `number`. `number`s can be fractional, whereas `integer`s cannot. So I've gone through and updated the places where we either do not return fractionals or cannot allow fractionals.